app/helpers/translatable_form_helper.rb

```diff
@@ -0,0 +1,50 @@
+module TranslatableFormHelper
+  def translatable_form_for(record, options = {})
```

Comment:
I took that code and naming straight from the [`form_for` source](https://github.com/rails/rails/blob/master/actionview/lib/action_view/helpers/form_helper.rb#L430), but I agree it's not very clear.

The array form doesn't necessarily represent a collection of records, but the "path" or scope to the record of interest e.g. `form_for([:admin, @budget, @investment], ...)`. I'll change it to `record_or_record_path` unless you suggest otherwise.

Reply:
Your suggestion is definitely better than mine. It looks like the Rails guys need us to do code reviews for them :stuck_out_tongue:!